sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -83,8 +83,11 @@ import {
   SubjectAlternativeNamesAll,
   CertificatePolicyProperties,
   PolicySubjectProperties,
-  DefaultCertificatePolicy
+  DefaultCertificatePolicy,
+  CertificateClientOptions,
+  LATEST_API_VERSION
```

Comment:
Ah, I see now. Just to import here, but not exported.

Reply:
Exactly! 🌞